app/controllers/application_controller/ci_processing.rb

```diff
@@ -952,25 +952,17 @@ def process_elements(elements, klass, task, display_name = nil, order_field = ni
       description = get_record_display_name(elem)
       name        = elem.send(order_field.to_sym)
       if task == "destroy"
-        audit = {:event        => "#{klass.name.downcase}_record_delete",
-                 :message      => "[#{name}] Record deleted",
-                 :target_id    => id,
-                 :target_class => klass.base_class.name,
-                 :userid       => session[:userid]}
-      end
-
-      model_name = ui_lookup(:model => klass.name)  # Lookup friendly model name in dictionary
-      begin
-        elem.send(task.to_sym) if elem.respond_to?(task)    # Run the task
-      rescue => err
-        add_flash(_("%{model} \"%{name}\": Error during '%{task}': ") % {:model => model_name, :name => description, :task => (display_name || task)} << err.message,
-                  :error)
+        process_element_destroy(elem, klass, name)
       else
-        if task == "destroy"
-          AuditEvent.success(audit)
-          add_flash(_("%{model} \"%{name}\": Delete successful") % {:model => model_name, :name => description})
+        model_name = ui_lookup(:model => klass.name) # Lookup friendly model name in dictionary
+        begin
+          elem.send(task.to_sym) if elem.respond_to?(task) # Run the task
+        rescue => err
+          add_flash(_("%{model} \"%{name}\": Error during '%{task}': ") %
+                   {:model => model_name, :name => description, :task => (display_name || task)} << err.message, :error)
```

Comment:
Oh haha...I didn't see the comment a few lines down that is similar... nevermind @mzazrivec :smiley: